cms/djangoapps/contentstore/tasks.py

```diff
@@ -397,10 +397,12 @@ def import_olx(self, user_id, course_key_string, archive_path, archive_name, lan
         user = User.objects.get(pk=user_id)
     except User.DoesNotExist:
         with translation_language(language):
+            LOGGER.error('Course import %s: Unknown User ID: {0}', courselike_key, user_id)
```

Comment:
Let's prefix it as : `CourseImport: %s ....

Reply:
LOGGER.error(f'CourseImport: {courselike_key}: Unknown User ID: {user_id}')

Reply:
That would make it difficult to debug on splunk. Let us keep it generic so that we can just put in one query on splunk and get all the related logs for the course import. Even info logs are useful here